src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/LoggerScenario.cs

```diff
@@ -73,11 +74,15 @@ public static async Task ExecuteAsync(InvocationContext context)
 
                 // The log entries above may get stuck in buffers in the runtime eventing infra or
                 // in the trace event library event processor due to their close proximity in being emitted.
-                // To mitigate this, wait a short time and send another log entry, which will cause the buffer
-                // to flush the existing entries. This log entry should be ignored by the logs tests.
-                await Task.Delay(CommonTestTimeouts.EventSourceBufferAvoidanceTimeout);
+                // To mitigate this, repeatedly wait a short time and send another log entry, which will cause the buffer
+                // to flush the existing entries. These log entry should be ignored by the logs tests.
+                ILogger flushCategory = loggerFactory.CreateLogger(TestAppScenarios.Logger.Categories.FlushCategory);
+                for (int i = 0; i < 3; i++)
```

Comment:
Any reason specifically for using 3 here (and if so, should it be documented)?

Reply:
I reduced it down to two and further clarified why that number was chosen.